### PR TITLE
Feature 2 added

### PR DESCRIPTION
**Pull request by rajoyraj**

### CODE REVIEW

File: Java/src/test/java/quintrix/Assignment/Bicycle.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package quintrix.Assignment;
+
+public class Bicycle extends Vehicle {
+
```
Comment (dentaylor) [inline Java/src/test/java/quintrix/Assignment/Bicycle.java:4]: Remove whitespace

File: Java/src/test/java/quintrix/Assignment/Bicycle.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package quintrix.Assignment;
+
+public class Bicycle extends Vehicle {
+	
+
+	private int odometer = 2000;
```
Comment (dentaylor) [inline Java/src/test/java/quintrix/Assignment/Bicycle.java:6]: Why 2000?

File: Java/src/test/java/quintrix/Assignment/Bicycle.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+	
+
+	private int odometer = 2000;
+
+	public int getOdometer() {
+
+		return this.odometer;
+	}
+
+	public void drive() {
```
Comment (dentaylor) [inline Java/src/test/java/quintrix/Assignment/Bicycle.java:13]: Pass a parameter to determine how far to drive.

File: Java/src/test/java/quintrix/Assignment/Bicycle.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+		this.odometer++;
+	}
+
+	public void start() {
+
```
Comment (dentaylor) [inline Java/src/test/java/quintrix/Assignment/Bicycle.java:19]: Remove whitespace

File: Java/src/test/java/quintrix/Assignment/BicycleTests.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+	@Test
+	public void canRideBicycle() {
+
+		Bicycle myBicycle = new Bicycle();
+		int currentOdometer = myBicycle.getOdometer();
+
+		myBicycle.drive();
+
+		Assert.assertEquals(currentOdometer + 1, myBicycle.getOdometer());
+
```
Comment (dentaylor) [inline Java/src/test/java/quintrix/Assignment/BicycleTests.java:18]: Remove whitespace

File: Java/src/test/java/quintrix/Assignment/Car.java [hunk 6]
```diff
@@ -1,6 +1,6 @@
 package quintrix.Assignment;
 
-public class Car {
+public class Car extends Vehicle {
 
 	String model;
 	private int odometer = 7000;
```
Comment (dentaylor) [inline Java/src/test/java/quintrix/Assignment/Car.java:6]: Why 7000?

**Comment by dentaylor** [comment]
I cannot accurately place a grade on the project if it is submitted after we reviewed it.

**Review by dentaylor — APPROVED** [review]

**Comment by rajoyraj** [comment]
Hi Sir,
    I had submitted it earlier but then i had deleted the branch since my git was not working properly. Then i had to resubmit it after creating a new branch.

Regards 
Rajoy

**Review by dentaylor — APPROVED** [review]

**Pull request merged**